bower.json

```diff
@@ -1,22 +1,25 @@
 {
   "name": "device-orientation",
-  "description": "Polymer element to detecting device orientation",
+  "description": "Polymer element to detecting device orientation and motion",
   "authors": [
     "Abdón Rodríguez Davila <a@abdonrd.com>"
   ],
   "keywords": [
     "web-component",
+    "webcomponent",
     "polymer",
     "device",
+    "position",
     "orientation",
+    "motion",
     "sensor"
   ],
-  "license": "https://abdonrd.github.io/LICENSE.txt",
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/abdonrd/device-orientation.git"
   },
-  "main": "device-orientation.html",
+  "main": "device-orientation-all.html",
```

Comment:
I believe this is not a good idea, because in this case `device-motion` is not displaying. See:

![screenshot](https://user-images.githubusercontent.com/7892779/27962184-c31bc9f4-6339-11e7-8dee-cd5cf73ac95e.png)

If we use `device-orientation-all.html` all is ok. Maybe revert? I can rename it to `device-orientation-elements.html`.